### PR TITLE
EASYOPAC-1115 - Added font, variables and other config files.

### PR DESCRIPTION
**Pull request by mihaelad**

### CODE REVIEW

File: scripts/kulture.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+    link.append('<span class="arrow"></span>');
+
+    var accordionBody = listContainerRef.find('.accordion-wrapper > .list-items');
+    accordionBody.attr("id", "news-library-filter-accordion");
+    accordionBody.attr("class", "list-items accordion-filter-body collapse");
+  }
+
+
+  $(document).ready(function () {
```
Comment (NicolaeGrigoriu) [inline scripts/kulture.js]: Here can be used drupal behaviours: https://www.drupal.org/node/756722#behaviors

File: scripts/kulture.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        $(this).removeClass('input-has-date');
+      } else {
+        $(this).addClass('input-has-date');
+      }
+    });
+  
+  });
+
+})(jQuery);
+
```
Comment (NicolaeGrigoriu) [inline scripts/kulture.js]: based on drupal coding standards, add new line at the end of the file

File: scss/components/_event-details.scss [hunk 3]
```diff
@@ -0,0 +1,252 @@
+.event-details-wrapper {
```
Comment (NicolaeGrigoriu) [inline scss/components/_event-details.scss]: add proper indendation for all scss files.

File: scss/mixins/_checkbox.scss [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      &:hover {
+        box-shadow: map-get($shadows, level-2);
+      }
+    }
+  }
+}
+
+.navbar-light .navbar-toggler {
+  border: none;
+}
```
Comment (NicolaeGrigoriu) [inline scss/mixins/_checkbox.scss]: add new line at the end of the file.

File: scripts/kulture.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+    link.append('<span class="arrow"></span>');
+
+    var accordionBody = listContainerRef.find('.accordion-wrapper > .pane-content');
+    accordionBody.attr("id", "category-filter-accordion");
+    accordionBody.attr("class", "accordion-filter-body collapse");
+  }
+
+  function addCollapseToNewsFilter(listContainerRef) {
+    console.log(listContainerRef);
```
Comment (NicolaeGrigoriu) [inline scripts/kulture.js]: is this line necessary?

**Review by NicolaeGrigoriu — CHANGES_REQUESTED** [review]
There are some changes to be made in the scss files and js

File: scripts/materials.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
+(function ($) {
+  "use strict";
+
+  Drupal.behaviors.ding_ting_materials = {
+    attach: function (context, settings) {
+
+      console.log($('.field-type-ting-reference > .field-items', context),'---')
```
Comment (NicolaeGrigoriu) [inline scripts/materials.js]: is this needed?

File: scss/components/_library-arrangementer.scss [hunk 7]
```diff
@@ -0,0 +1,584 @@
+.page-bibliotek .layout-wrapper {
+  margin-top: 70px;
+
+  .secondary-content {
+    .panel-pane.pane-menu-title {
+      display: none;
+    }
+
+    .pane-og-menu-og-single-menu-block h2 a {
```
Comment (NicolaeGrigoriu) [inline scss/components/_library-arrangementer.scss]: is scss file shouldn't this single selector?

File: scss/components/_library-details.scss [hunk 8]
```diff
@@ -0,0 +1,446 @@
+#ding-library-front {
+  margin-top: 70px;
+  padding: 0;
+
+  .secondary-content {
+
+    .sub-menu-title a,
+    .sub-menu-wrapper .pane-title a,
+    .panel-pane-inner h2.pane-title a {
```
Comment (NicolaeGrigoriu) [inline scss/components/_library-details.scss:8]: shouldn't here be a single selector in scss file?

File: scss/fonts/_fortuna.scss [hunk 9]
```diff
@@ -0,0 +1,11 @@
+@font-face {
+  font-family: 'Fortuna Regular';
+  // src     : url('../fonts/Fortuna Regular.eot');
+  src:
+    //  url('../fonts/Fortuna Regular.eot?#iefix') format('embedded-opentype'),
+    //   url('../fonts/Fortuna Regular.woff') format('woff'),
+    url('../fonts/Fortuna Regular.ttf') format('truetype');
+  //   url('../fonts/Fortuna Regular.otf') format('opentype');
```
Comment (NicolaeGrigoriu) [inline scss/fonts/_fortuna.scss:8]: are the comments required?

File: scss/fonts/_rubik.scss [hunk 10]
```diff
@@ -0,0 +1,47 @@
+@font-face {
+  font-family: 'Rubik-Regular';
+  //src      : url('../../fonts/Rubik-Regular.eot');
+  src:
+    url('../fonts/Rubik-Regular.ttf') format('truetype');
+  //   url('../fonts/Rubik-Regular.eot?#iefix') format('embedded-opentype'),
+  // url('../fonts/Rubik-Regular.woff') format('woff'),
+  // url('../fonts/Rubik-Regular.otf') format('opentype');
```
Comment (NicolaeGrigoriu) [inline scss/fonts/_rubik.scss:8]: are the comments required?

File: templates/node/node--ding-event.tpl.php [hunk 11]
```diff
@@ -0,0 +1,164 @@
...
+        <?php print render($content['group_left']['field_ding_event_title_image']); ?>
+      </div>
+      <div class="right col-lg-7">
+        <div class="info-top">
+          <i class="tag-icon"></i>
+          <?php print render($content['field_ding_event_category']); ?>
+        </div>
+        
+        <h1><?php print $title; ?></h1>
+
```
Comment (NicolaeGrigoriu) [inline templates/node/node--ding-event.tpl.php]: are the new lines needed in this tpl file?

**Review by NicolaeGrigoriu — CHANGES_REQUESTED** [review]

File: scss/components/_library-details.scss (hunk 8 above)
Comment (mihaelad) [reply scss/components/_library-details.scss:8]: On the culture site are 3 different menu but the styles are identical and this why is need these selector

File: scss/fonts/_fortuna.scss (hunk 9 above)
Comment (mihaelad) [reply scss/fonts/_fortuna.scss:8]: To work the fonts in all of browsers are required all of this formats which are committed. I expect them from the customer.

File: scss/fonts/_rubik.scss (hunk 10 above)
Comment (mihaelad) [reply scss/fonts/_rubik.scss:8]: To work the fonts in all of browsers are required all of this formats which are committed. I expect them from the customer.

**Pull request merged**